au.yaml

```diff
@@ -517,7 +517,7 @@ tests:
     expect:
       name: "Queen's Birthday"
   - given:
-      date: ['2024-09-23', '2025-09-30']
+      date: ['2025-09-30']
```

Comment:
How come this one was changed? It's still relevant for 2024?

Reply:
I keep reading it in reverse, will restore the test